bylaws.tex

```diff
@@ -0,0 +1,288 @@
+
+The Directors, all officers of the Corporation, and all persons who are alumni in good standing of the Eta Tau Chapter of Beta Theta Pi Fraternity shall be permitted to attend any regular meeting of the Board of Directors. Other persons may be, but need not be, excluded.
+
+\section{Special Meetings}
+
+Special meetings of the Board of Directors may be called by order of the Chairman of the Board of Directors or the President, and shall be held at such time and place as shall be designated by the Chairman or President in the notice of the meeting.
+
+The Directors and all officers of the Corporation shall be permitted to attend any special meeting of the Board of Directors. Other persons may be, but need not be, excluded.
+
+\section{Electronic Participation in Meetings}
+
+The Board of Directors may permit any or all Directors to participate in a regular or special meeting by, or conduct the meeting through the use of, any means of communication by which all Directors participating may simultaneously hear each other during the meeting. A Director participating in a meeting by this means is considered to be present in person at the meeting, and is said to be `participating electronically'.
+
+\section{Notice of Meetings}
+
+Notice of the time and place of any meeting of the Board of Directors shall be delivered personally, by mail, or by email, to each of the elected Directors; and shall be delivered to the ex officio Directors collectively by mail or email to such mailing address or email address as the Eta Tau Chapter of Beta Theta Pi Fraternity shall designate for receiving such notice. Notice shall be delivered at least seven days prior if the Directors are permitted to participate electronically, and at least fourteen days prior if any Director is not permitted to participate electronically. A Director may waive notice. The attendance of a Director at a meeting shall constitute a waiver of notice of that meeting, except where a Director attends a meeting for the purpose of objecting to the transaction of business because the meeting is not lawfully called or convened. Neither the business to be transacted at, nor the purpose of, any regular or special meeting of the Board of Directors need be specified in the notice of the meeting unless required by statute.
+
+\section{Quorum}
+
+Four Directors present in person shall constitute a quorum for the transaction of business at any meeting of the Board of Directors, unless the presence of a greater number is required by law, in which case the quorum shall be such number as the law requires.
```

Comment:
Does there need to be a clause if there aren't 4 directors on the board?

Reply:
There will always be at least three Directors because of the three ex officio Directors. If there are less than four Directors on the Board that would mean there are zero elected Directors. I assume that won't happen, especially because the Board isn't allowed to do anything but elect Directors once the number of elected Directors gets less than four. So it reaching zero would be pretty difficult I imagine.

Reply:
As long as there isn't conflicting bylaws that require quorum for new directors this is fine

Reply:
A quorum is required in order to elect new directors. But the only way a quorum of four would be unobtainable would be in the event that there are zero elected directors left, leaving only the three ex officio directors (who can't resign). Given that the bylaws also say the following, my reasoning is that it would be highly improbable to end up in the situation where you can't obtain quorum.

> In the event that the number of elected Directors falls to less than four, through resignations or any other cause, the Board of Directors shall take no action other than the election of Directors until the number of elected Directors is at least four.

In other words if the total number of directors falls to less than seven the board must elect more before it does anything else. In order to hit the scenario where you can't obtain quorum, all of the elected directors would have to resign at once. Because you seem to still be concerned about that scenario, I'll make an adjustment so that if the number of directors is less than four then quorum will be three. But that should never happen.

Reply:
The difference between "directors", "elected directors", and "ex officio directors" is significant.

Reply:
Just fixed it in 98d193958059c06ce4c54c4e612c55e1cbc8d259 so that even if all of the elected directors resign, the three ex officio directors will still be able to elect more directors.